src/index.html

```diff
@@ -1,14 +1,50 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="/home" class="logo__img"></a>
+      </div>
+      <nav class="navbar">
```

Comment:
Let's add some spaces between child elements in HTML.
```suggestion
      <div class="logo">
        <a href="/home" class="logo__img"></a>
      </div>
      
      <nav class="navbar">
```